client/landing/gutenboarding/components/signup-form/index.tsx

```diff
@@ -101,46 +102,55 @@ const SignupForm = ( { onRequestClose }: Props ) => {
 
 	return (
 		<Modal
-			className="signup-form"
-			title={ NO__( 'Sign up to save your changes' ) }
+			className={ 'signup-form' }
+			title={ NO__( 'Save your progress' ) }
 			onRequestClose={ closeModal }
 			focusOnMount={ false }
-			isDismissible={ ! isFetchingNewUser }
+			isDismissible={ false }
+			overlayClassName={ 'signup-form__overlay' }
 			// set to false so that 1password's autofill doesn't automatically close the modal
 			shouldCloseOnClickOutside={ false }
 		>
-			<form onSubmit={ handleSignUp }>
-				<TextControl
-					label={ NO__( 'Your Email Address' ) }
-					value={ emailVal }
-					disabled={ isFetchingNewUser }
-					type="email"
-					onChange={ setEmailVal }
-					placeholder={ NO_x(
-						'E.g., yourname@email.com',
-						"An example of a person's email, use something appropriate for the locale"
-					) }
-					required
-					autoFocus={ true } // eslint-disable-line jsx-a11y/no-autofocus
-				/>
-				{ errorMessage && (
-					<Notice className="signup-form__error-notice" status="error" isDismissible={ false }>
-						{ errorMessage }
-					</Notice>
-				) }
-				<div className="signup-form__footer">
-					<p className="signup-form__terms-of-service-link">{ tos }</p>
-
-					<ModalSubmitButton disabled={ isFetchingNewUser } isBusy={ isFetchingNewUser }>
-						{ NO__( 'Create your account' ) }
-					</ModalSubmitButton>
-				</div>
-			</form>
-			<div className="signup-form__login-links">
-				<Button isLink href={ '/log-in?redirect_to=' + encodeURIComponent( loginRedirectUrl ) }>
-					{ NO__( 'Log in to create a site for your existing account.' ) }
-				</Button>
-			</div>
+			<SignupFormHeader
+				onRequestClose={ closeModal }
+				loginUrl={ '/log-in?redirect_to=' + encodeURIComponent( loginRedirectUrl ) }
+			/>
+
+			<main className="signup-form__body">
```

Comment:
I don't think `<main>` is appropriate here. You're only allowed one on the page at a time, and it would normally be the non-header/footer, non-modal, main content of the page.

We're not getting an a11y violation warning in the browser, because Gutenboarding doesn't already have a `<main>` element, but it probably should.

You might have been looking for something like `<dialog>` or `<div role="dialog">`, but never fear, `<Modal>` has already taken care of that for you. So I think a `<div>` would be fine in this case.

Reply:
Oh, thanks — yes, `dialog` is more in the direction of what I was thinking. Thanks for looking into this, I'll switch to a `div`!